frontend/templates/troubleshooting/index.tsx

```diff
@@ -133,22 +136,26 @@ const Wiki: NextPageWithLayout<{
                display="grid"
                sx={{
                   gridTemplateColumns: {
-                     base: '[toc] none [wrapper] 1fr',
+                     base: '[toc] 0 [wrapper] 1fr',
                      lg: '[toc] auto [wrapper] 1fr',
                   },
                }}
                ref={contentContainerRef}
             >
-               <TOC
-                  className="summary"
-                  contentRef={contentContainerRef}
-                  borderRight={{ lg: '1px solid' }}
-                  borderColor={{ lg: 'gray.300' }}
-                  maxWidth={{ base: 'calc(100% + 2 * var(--chakra-space-4))' }}
-                  listItemProps={{ paddingLeft: { lg: 4 } }}
-                  gridArea="toc"
-                  width="220px"
-               />
+               {isDesktop && (
```

Comment:
I disagree with this change.

`useBreakpoint` will wait until after js is run client side, this means you can get CLS.

I had worked around this inside `<TOC>` here:

https://github.com/iFixit/react-commerce/blob/vulcan--1.1-style-refactor/frontend/templates/troubleshooting/toc.tsx#L52

While I allowed the mobile TOC to wait until client side since it fades in:
https://github.com/iFixit/react-commerce/blob/vulcan--1.1-style-refactor/frontend/templates/troubleshooting/toc.tsx#L102

Reply:
Wait this will hide the mobile TOC all together. I don't think this is intended

Reply:
Yeah, today is not my favorite day:

> Hang on: I see some issue. Apologies, I'm having to use the github.com UI to make changes temporarily 😞

Reply:
It's tricky there.

I am actually rendering both components (mobile and desktop) always and swapping the display value of the desktop (via CSS props) and then the mobile one is just an empty div until this line is true:

https://github.com/iFixit/react-commerce/blob/vulcan--1.1-style-refactor/frontend/templates/troubleshooting/toc.tsx#L130

`const isVisible = wantsMobile && showMobileTOC && activeItem;`

Where:
* `wantsMobile` is a clientside mobile query check
* `showMobileTOC` is a check that we have scrolled past the main content container
* `activeItem` is that there is something the TOC considers as the visible element

Reply:
Gotcha. I should be able to adjust the grid to suit. It's a little tricky since it's the child elements that have `display` applied.